modules/telemetry/include/hephaestus/telemetry/measure_sink.h

```diff
@@ -0,0 +1,30 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include <chrono>
+#include <string>
+
+namespace heph::telemetry {
+using ClockT = std::chrono::system_clock;
+
+struct MeasureEntry {
+  [[nodiscard]] auto operator==(const MeasureEntry& other) const -> bool = default;
+
+  std::string component;  ///< The component that is logging the measure, e.g. SLAM, Navigation, etc.
+  std::string tag;        ///< The tag of the measure used to identify who created it, e.g. "front_camera",
+                          ///< "motor1", etc.
+  ClockT::time_point measure_timestamp;
+  std::string json_values;
+};
```

Comment:
Q: what about ids? How do you associate multiple `MeasureEntries` with each other?
Timestamp works, but it's prone to errors if the timestamps are not 100% identical

Reply:
Could please make an example?

Reply:
Example:

Synchronized data is produced with timestamp t0. SLAM consumes this data and produces a world pose with timestamp t0.

Planner module consumes the map and SLAM pose at timestamp t0 and plans a trajectory. 

Both modules output telemetry data, and the backend associates the SLAM telemetry with the planner's telemetry.

I don't suggest to add an id immediately, as this is a hypothetical discussion at this point. The question I would like to raise is how to associate telemetry data logged by different modules?

Reply:
ok I see, it is a good point. I don't see a downside in having an ID field as a `std::size_t` value. In influxdb it can set as a tag.

Reply:
+1 for the ids

Reply:
Ok, let's add an `id` field then. As said, I don't have an immediate use case, but similar to sensor data, or Perfetto tracing, if we want to associate ("synchronize") metrics later, an id is much easier to work with than timestamps.